src/pyopticon/_system/_automation_widget.py

```diff
@@ -380,6 +386,17 @@ def _load_automation_file(self): # Open a dialog and load a file for automation
             self.time_to_go_readout.set(str(timedelta(seconds=self.time_to_go)))
         print("Loaded "+filename)
 
+
+    def update(self, event):
+        """When a notification is sent by the publisher (dashboard) the event is checked. If the event has 'New automation file' 
+        in it the existence of an automation file in the Recipe database will be checked, and if it does exist it will be uploaded.
+        
+        :param event: The event that just occurred
+        :type event: String
+        """
+        if 'New automation file' in event:
```

Comment:
Let's factor this string out into a constant, so that it's more generic and can be reused by other potential clients. I'm thinking something like 
`PYOPTICON_DASHBOARD_EVENT_NEW_AUTOMATION_FILE = "New Automation File"`
That way we can avoid typos in strings and leverage symbol autocompletion, etc.